engine/include/cubos/engine/renderer/plugin.hpp

```diff
@@ -25,11 +25,12 @@ namespace cubos::engine
         float zFar;  ///< The far clipping plane.
     };
 
-    /// Resource which identifies the entity which is the current active camera.
-    /// The entity referenced by this resource must have a `Camera` component.
-    struct ActiveCamera
+    /// @brief Resource which identifies the camera entities to be used by the renderer.
+    struct ActiveCameras
     {
-        core::ecs::Entity entity; ///< The entity which is the active camera.
+        /// @brief Entities which represent the current active cameras. If more than one is set,
+        /// the renderer automatically splits the screen.
+        core::ecs::Entity entities[2];
```

Comment:
We just need two cameras for MOJO, but this should be eventually expanded to account for greater amounts or be well documented that only two are supported.
If that work will be done after this being merged, add a comment in the code and open an issue.